spark-jobs/src/test/scala/za/co/absa/enceladus/standardization/StandardizationFixedWidthSuite.scala

```diff
@@ -30,7 +30,7 @@ import za.co.absa.enceladus.utils.implicits.DataFrameImplicits.DataFrameEnhancem
 import za.co.absa.enceladus.utils.udf.UDFLibrary
 
 class StandardizationFixedWidthSuite extends FunSuite with SparkTestBase with MockitoSugar{
-  private implicit val udfLibrary:UDFLibrary = new UDFLibrary()
+  private implicit val udfLibrary: UDFLibrary = new UDFLibrary()
   private val log: Logger = LoggerFactory.getLogger(this.getClass)
```

Comment:
**Original code issue:** In case if you would be fixing the long lines or anything else, the `val log: Logger` is not used.